src/components/pages/account/InvestmentChart.tsx

```diff
@@ -128,6 +123,13 @@ export default function InvestmentChart({
           tooltip: {
             backgroundColor: '#323b44',
             callbacks: {
+              title: (tooltipItems) => {
+                if (tooltipItems[0].datasetIndex === 0) {
+                  return dates[tooltipItems[0].dataIndex].toFormat('DD');
```

Comment:
this is kind of a hack. We draw the bar in the 1st of the month to align it with the tick but in reality we've calculated at the last day of the month.